MAVProxy/tools/graphs/ekf3Graphs.xml

```diff
@@ -1,272 +1,272 @@
     <expression>XKF3[1].IMX XKF3[1].IMY XKF3[1].IMZ XKF3[1].IYAW:2</expression>
   </graph>
 
-  <graph name='EKF3/Mag Innovations Lane3'>
+  <graph name='EKF3/Innovations/Mag Innovations Lane3'>
     <description>EKF3 Mag Innovations Lane3</description>
     <expression>XKF3[2].IMX XKF3[2].IMY XKF3[2].IMZ XKF3[2].IYAW:2</expression>
   </graph>
   
-  <graph name='EKF3/Euler Roll'>
+  <graph name='EKF3/Solution Values/Euler Roll'>
     <description>EKF3 Euler Roll</description>
     <expression>ATT.Roll XKF1[0].Roll XKF1[1].Roll XKF1[2].Roll</expression>
     <expression>ATT.Roll XKF1[0].Roll XKF1[1].Roll</expression>
     <expression>ATT.Roll XKF1.Roll</expression>
   </graph>
 
-  <graph name='EKF3/Euler Pitch'>
+  <graph name='EKF3/Solution Values/Euler Pitch'>
     <description>EKF3 Euler Pitch</description>
     <expression>ATT.Pitch XKF1[0].Pitch XKF1[1].Pitch XKF1[2].Pitch</expression>
     <expression>ATT.Pitch XKF1[0].Pitch XKF1[1].Pitch</expression>
     <expression>ATT.Pitch XKF1.Pitch</expression>
   </graph>
 
-  <graph name='EKF3/Euler Yaw'>
+  <graph name='EKF3/Solution Values/Euler Yaw'>
     <description>EKF3 Euler Yaw</description>
     <expression>ATT.Yaw XKF1[0].Yaw XKF1[1].Yaw XKF1[2].Yaw</expression>
     <expression>ATT.Yaw XKF1[0].Yaw XKF1[1].Yaw</expression>
     <expression>ATT.Yaw XKF1.Yaw</expression>
   </graph>
   
-  <graph name='EKF3/Velocity N'>
+  <graph name='EKF3/Solution Values/Velocity N'>
     <description>EKF3 Velocity North</description>
     <expression>XKF1[0].VN XKF1[1].VN XKF1[2].VN</expression>
     <expression>XKF1[0].VN XKF1[1].VN</expression>
     <expression>XKF1.VN</expression>
   </graph>
 
-  <graph name='EKF3/Velocity E'>
+  <graph name='EKF3/Solution Values/Velocity E'>
     <description>EKF3 Velocity East</description>
     <expression>XKF1[0].VE XKF1[1].VE XKF1[2].VE</expression>
     <expression>XKF1[0].VE XKF1[1].VE</expression>
     <expression>XKF1.VE</expression>
   </graph>
 
-  <graph name='EKF3/Velocity D'>
+  <graph name='EKF3/Solution Values/Velocity D'>
     <description>EKF3 Velocity Down</description>
     <expression>XKF1[0].VD XKF1[1].VD XKF1[2].VD</expression>
     <expression>XKF1[0].VD XKF1[1].VD</expression>
     <expression>XKF1.VD</expression>
   </graph>
 
-  <graph name='EKF3/Velocity dDP'>
+  <graph name='EKF3/Solution Values/Velocity dDP'>
     <description>EKF3 Velocity Down (DP derivative)</description>
     <expression>XKF1[0].dPD XKF1[1].dPD XKF1[2].dPD</expression>
     <expression>XKF1[0].dPD XKF1[1].dPD</expression>
     <expression>XKF1.dPD</expression>
   </graph>
 
-  <graph name='EKF3/Accel Bias Lane1'>
+  <graph name='EKF3/Solution Values/Accel Bias Lane1'>
     <description>EKF3 Accel Bias Lane1</description>
     <expression>XKF2[0].AX XKF2[0].AY XKF2[0].AZ</expression>
     <expression>XKF2.AX XKF2.AY XKF2.AZ</expression>
   </graph>
 
-  <graph name='EKF3/Accel Bias Lane2'>
+  <graph name='EKF3/Solution Values/Accel Bias Lane2'>
```

Comment:
I'd prefer /State/ instead of /Solution Values/

Reply:
done